Assets/Xbox Live/Scripts/Stats/StatBase.cs

```diff
@@ -34,6 +38,37 @@ private void Awake()
         // Ensure that a StatsManager has been created so that stats will be sync with the service as they are modified.
         var statsManager = StatsManagerComponent.Instance;
     }
+
+    void Start()
+    {
+        if (XboxLiveUserHelper.Instance.SingleUserModeEnabled)
+        {
+            this.XboxLiveUser = XboxLiveUserHelper.Instance.SingleXboxLiveUser;
+            if (this.XboxLiveUser == null || this.XboxLiveUser.User == null || !this.XboxLiveUser.User.IsSignedIn)
+            {
+                StatsManagerComponent.Instance.LocalUserAdded += (sender, args) =>
+                {
+                    this.HandleGetStat(args.User, this.Name);
+                };
+            }
+        }
+    }
+
+    protected void Update()
+    {
+        if (this.XboxLiveUser != null && this.XboxLiveUser.User != null && this.XboxLiveUser.User.IsSignedIn && !this.isLocalUserAdded)
+        {
```

Comment:
Probably worth moving !this.LocalUserAddedSetup check first.

---
In reply to: [119976333](https://github.com/Microsoft/xbox-live-unity-plugin/pull/66#discussion_r119976333) [](ancestors = 119976333)